src/components/coreComponents/Table.js

```diff
@@ -31,6 +31,17 @@ class CustomTable extends React.Component {
               </tr>
             ))}
         </tbody>
+          <tbody>
+          {this.state.interviews &&
```

Comment:
to avoid multiple checks may you can initialize the object with an empty parameter:

`state = {
interview:{body:[]}
}`